routes.js

```diff
@@ -8,6 +8,7 @@
 //
 const fs = require("fs");
 const https = require("https");
+const dotenv = require("dotenv");
```

Comment:
Good idea. At Glitch you don't need it but ppl that will install it will need dotenv pkg.